torch/csrc/jit/codegen/cuda/iter_visitor.cpp

```diff
@@ -26,9 +28,37 @@ std::vector<Statement*> IterVisitor::next(Val* v) {
   return {};
 }
 
-std::vector<Statement*> IterVisitor::next(Expr* expr) {
+std::vector<Statement*> IterVisitor::next(Expr* expr, bool respect_compute_at) {
   FusionGuard::getCurFusion()->assertInFusion(expr, "Cannot traverse expr, ");
-  return {expr->inputs().begin(), expr->inputs().end()};
+  std::vector<Statement*> next_stmts{expr->inputs().begin(),
+                                     expr->inputs().end()};
+  // TODO: Remove the temporary getenv
+  if (std::getenv("FUSER_SORT_INPUT_EXPRS") && respect_compute_at) {
+    if (expr->output(0)->getValType().value() == ValType::TensorView) {
```

Comment:
Could you add an error for `outputs().size() > 1`? Otherwise I think we should check all TVs if there's multiple outputs.

Reply:
Added.